pkg/operator/ceph/csi/spec.go

```diff
@@ -683,3 +736,119 @@ func (r *ReconcileCSI) validateCSIVersion(ownerInfo *k8sutil.OwnerInfo) (*CephCS
 	}
 	return version, nil
 }
+
+func (r *ReconcileCSI) configureHolder(enabledDrivers []driverDetails, tp templateParam, pluginTolerations []corev1.Toleration, pluginNodeAffinity *corev1.NodeAffinity) error {
+	for _, multusCluster := range r.multusClusters {
+		for _, driver := range enabledDrivers {
+			cephPluginHolder, err := templateToDaemonSet("cephpluginholder", driver.holderTemplate, tp)
+			if err != nil {
+				return errors.Wrapf(err, "failed to load ceph %q plugin holder template", driver.fullName)
+			}
+
+			// DO NOT set owner reference on ceph plugin holder daemonset, this DS must never restart unless
+			// the entire node is rebooted
+			holderPluginTolerations := getToleration(r.opConfig.Parameters, driver.toleration, pluginTolerations)
```

Comment:
setting pod label is missing?

Reply:
labels are inherited from `CSIRBDPodLabels` and `CSICephFSPodLabels`. Am I missing something?

Reply:
Yes correct I missed this one

Reply:
No worries, thanks.